### PR TITLE
Use plurals on task names

### LINKED ISSUE #33: Allow continue when inconsistency is found
We need to add the feature of continue even if we found an inconsistency between the benchmarks, logging a warning if it's found and compare only the common rows

### PR DESCRIPTION
**Pull request by AdrianRaFo**
Fixes #33

### CODE REVIEW

**Comment by AdrianRaFo** [comment]
Microsite is done and ready for lang review @MaureenElsberry @jkmcclellan

**Comment by MaureenElsberry** [comment]
Hey @AdrianRaFo - I see that the homepage is done, but what is the plan for the documentation - it's all lorem ipsum still... I would prefer we wait to do the language review until all components are available for editing at https://47deg.github.io/hood/.

**Comment by AdrianRaFo** [comment]
Oh, @MaureenElsberry this PR is where I'm filling the microsite

**Comment by MaureenElsberry** [comment]
Ah, gotcha, sorry - I haven't drank coffee yet - went directly to the microsite... @jkmcclellan - if you can review the lang on these docs that would be great

**Comment by jkmcclellan** [comment]
I've finished adding my lang edits, @MaureenElsberry.

File: deploy-scripts/deploy.sh [hunk 1]
```diff
@@ -7,9 +7,9 @@ if [ "$TRAVIS_BRANCH" == "master" ]; then
     if [[ "$VERSION_NAME" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
         echo "Starting script for Release $VERSION_NAME"
         . $(dirname $0)/deploy_release.sh
-    elif [[ "$VERSION_NAME" == *-SNAPSHOT ]]; then
-        echo "Starting script for Snapshot Release $VERSION_NAME"
-        . $(dirname $0)/deploy_snapshot.sh
+#    elif [[ "$VERSION_NAME" == *-SNAPSHOT ]]; then
+#        echo "Starting script for Snapshot Release $VERSION_NAME"
+#        . $(dirname $0)/deploy_snapshot.sh
```
Comment (fedefernandez) [inline deploy-scripts/deploy.sh]: No snapshot no party

Comment (AdrianRaFo) [reply deploy-scripts/deploy.sh]: Snapshots never worked

Comment (fedefernandez) [reply deploy-scripts/deploy.sh]: okay

**Pull request merged**